### DISCUSSION TITLE
Training the supported models from sratch?

### DISCUSSION THREAD

**Discussion by ghoshsdeeep** [open-mmlab/mmclassification#1381] [General]
How is any supported model trained from scratch on on standard datasets  like VOG or ImageNet which are present inside mmcls/datasets?

**Comment by Ezra-Yu** [comment]
such as resnet, swin, vit, vgg, repvgg, mobileOne, convnext and so on.

**Reply by ghoshsdeeep** [reply]
Yes those models how do we train from scratch?
